samples/guestbook/main.go

```diff
@@ -279,26 +278,18 @@ func (app *application) sign(w http.ResponseWriter, r *http.Request) {
 // serveBlob handles a request for a static asset by retrieving it from a bucket.
 func (app *application) serveBlob(w http.ResponseWriter, r *http.Request) {
 	key := mux.Vars(r)["key"]
-	blobRead, err := app.bucket.NewReader(r.Context(), key, nil)
+	blobReader, err := app.bucket.NewReader(r.Context(), key, nil)
 	if err != nil {
 		// TODO(light): Distinguish 404.
```

Comment:
From [this comment](https://github.com/google/go-cloud/pull/1885/files#r277760225): we can also address this TODO.

Reply:
Done.